app/models/enrollment.rb

```diff
@@ -0,0 +1,12 @@
+class Enrollment < ApplicationRecord
+  belongs_to :student, class_name: "User"
+  belongs_to :course
+
+  enum completion_status: %i[not_started in_progress completed].freeze
+
+  validates :student_id, presence: true
+  validates :course_id, presence: true
```

Comment:
I don't think you need the validation as these are belongs to here.

Reply:
Removed the validations apu.